### ISSUE TITLE
Address USWDS contrast warnings

### ISSUE THREAD

**Issue by rahearn** [HHS/nih-oite-website#122]
Our USWDS build is emitting several warnings around color contrast values. It would be nice to clean these up so they don't become pa11y failures down the line.
